### PR TITLE
Twilson63/feat test hyper test 325

### PR DESCRIPTION
**Pull request by twilson63**
enhanced connect to support services request, as well as, updated test to call hyper connect info services.

## clean up 

Also did updated dependencies and set up lint and fmt test failures for all packages, finally extended testing surface area to 1.13.x and 1.14.x

### CODE REVIEW

File: .github/workflows/test-test.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: test hyper test
```
Comment (TillaTheHun0) [inline .github/workflows/test-test.yml:1]: Test the tests! Hehe :)

File: packages/test/mod.js [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
+  ]);
+}
 const hyperCS = answers.hyper === "" ? cs : answers.hyper;
 const hyper = connect(hyperCS)();
 
-const services = ['data', 'cache', 'search', 'storage', 'queue']
+// get services that are active on the hyper instance
+const services = await fetch(await hyper.info.services()).then((res) =>
+  res.json()
+).then((r) => r.services);
```
Comment (TillaTheHun0) [inline packages/test/mod.js:24]: Nice one

**Review by TillaTheHun0 — APPROVED** [review]
LGTM

File: .github/workflows/test-test.yml (hunk 1 above)
Comment (twilson63) [reply .github/workflows/test-test.yml:1]: Yeah, just felt good to have test for linting and fmt

**Pull request merged**